### ISSUE TITLE
Steam: Decide Release Name

### ISSUE THREAD

**Issue by RoyAwesome** [EmpiresCommunity/Unreal-Empires-Project#28]
The game name "Empires" is taken by some shitty survival game asset flip.  We should brainstorm some ideas for a release name.

We want to come up with names with "Empires" in the title, and also that reflect a wider war happening.  The general vibe is for this to be a Sequel to the Empires Mod, and thus the Northern Faction has successfully rebelled against the Brenodi Empire, and is pushing them back.  

In the end, the Empires team will select a name.  The point of this issue is to brainstorm ideas that sound good, and take them to the next stage of discussion.  

Please post one name idea per response.  If you like an idea, upvote it with a 👍 reaction.  If you really don't like one, give it a 👎

**Comment by RoyAwesome** [comment]
Ideas Under Consideration:

* Empires: Rise of Jekotia (My personal First Choice)
* Empires: Uprising (My personal Second Choice)
* Empires: New Dawn
* Empires: Northern Front
* Empires: Full Scale Warfare
* The Fall of Empires
* Empires: Age of Decline

**Comment by bloodwiper** [comment]
Empires: Jekotia Irredenta

**Comment by Xiaminou** [comment]
Minor detail, but, most people will not know how to pronounce Jekotia so I'd lean toward the second option:
Empires: Uprising

I still think naming the game just "Empires" is too vague and forgettable and there's too many games with Empires in their name.
At the very least I would advise naming it ""Something" Empires", for example, you could use the name of the world Empires takes place in.
And then you can name it ""Something" Empires: Uprising" if you want to.

**Comment by bloodwiper** [comment]
Factions and Empires

**Comment by bloodwiper** [comment]
Ember Pyres

**Comment by MayamaMayama** [comment]
Unreal empires is set some years (decades? a century?) after the original one so the new title should reflect that.

**Comment by empZealoth** [comment]
TBH none of the "new", "rise", "dawn" etc make sense given the lore that Jekotia is already an established thing in UEpires - those would fit original Empires
At least what I understood is that the UEpires lore says that this is way more of an even matchup. Jekotia is once again a full on state actor and BE is going into actual war footing

**Comment by slzimmer** [comment]
Maybe it might be worth spelling out what empires is in the name to draw new people in like:
Emipres: A unique hybrid rts/fps experience

**Comment by empZealoth** [comment]
> Maybe it might be worth spelling out what empires is in the name to draw new people in like: Emipres: A unique hybrid rts/fps experience

I personally firmly believe the game should stand on its own merits. Overfilling the description with subjective additives like "amazing", "unique" etc seems cringy to me. Putting it in the title to me is a cardinal sin. 

We can definitely use steam tags to mark it properly and inform people of the genre

**Comment by RoyAwesome** [comment]
> Minor detail, but, most people will not know how to pronounce Jekotia so I'd lean toward the second option: Empires: Uprising
> 
> I still think naming the game just "Empires" is too vague and forgettable and there's too many games with Empires in their name. At the very least I would advise naming it ""Something" Empires", for example, you could use the name of the world Empires takes place in. And then you can name it ""Something" Empires: Uprising" if you want to.

Yeah.  "Fallen Empires" comes to mind but there is nothing fallen about these empires.  We have to be careful not to trend to closely to "Age of Empires", because microsoft can and will go to war over anything that gets close.  

I do lean towards using either Jekotia or Brenodi in the title because it gives us a word that is extremely unique and google-able.

**Comment by RoyAwesome** [comment]
Empires: Pax Brenodi is another idea

**Comment by bloodwiper** [comment]
Are Brenodi or Jekotia really that hard of names for people to learn? I don't personally think so, but seems like a lot of big titles don't include any proper nouns.

**Comment by bloodwiper** [comment]
Empires: Bellicose Brenodi

**Comment by RoyAwesome** [comment]
Yeah, I don't think "Jekotia" is too hard of a word for people to learn or spell.   It's unique, interesting, and googleable.  

For what it's worth, I pronounce it "Jeh-ko-sha"

**Comment by empZealoth** [comment]
> Yeah, I don't think "Jekotia" is too hard of a word for people to learn or spell. It's unique, interesting, and googleable.
> 
> For what it's worth, I pronounce it "Jeh-ko-sha"

Time to start a thousand year war, but it's clearly pronounced like "Je" from jenkins, "ko" from coffee and "tia" from Celestia

As for the name, Empires: Jekotia Ascendant makes sense - the OG game was uprising, now it's actual war with Jekotia becoming fully capable state actor (at least as far as my understanding of currently proposed lore). Also, the name is IMO just the right amount of pretentious, unique enough to allow to serve as search keyword without being over the top

**Comment by bloodwiper** [comment]
I like it. Is the contemporary Jekotian state's territory solely comprised of land captured from the Brenodi Empire in the past conflicts, or did they always have a state? Are the Jekotians still irredentist or is it Brenodi revanchism that escalates the new war? Also thoughts on the OG spelling of Chekotia?

**Comment by RoyAwesome** [comment]
in my mind, the Jekotians mounted a successful rebellion in Empires, and were able to reclaim their lands.  However, the Brenodi, who's entire goal was one united planet, are **not** happy about that, and strike back, attempting to re-subjugate that last remaining holdout preventing Pax Breonodi.

Pretty simple motivations for both factions, and Jekotia Rising gives a really good anchor point for the storyline.

**Comment by MayamaMayama** [comment]
> We can definitely use steam tags to mark it properly and inform people of the genre

You mean like "addicting (not in a good way)" "miserable" "worse than dota"?

**Comment by empZealoth** [comment]
> in my mind, the Jekotians mounted a successful rebellion in Empires, and were able to reclaim their lands. However, the Brenodi, who's entire goal was one united planet, are **not** happy about that, and strike back, attempting to re-subjugate that last remaining holdout preventing Pax Breonodi.
> 
> Pretty simple motivations for both factions, and Jekotia Rising gives a really good anchor point for the storyline.

Thats why im stuck on Jekotia Ascendant - canonically it was in the OG empiresmod that Jekotians did all the uprising/rising from the ashes, now they are ascending to end the unipolar world order/end "Pax Brenodi". There is no peace to be had in the old order. Either Brenodi will have to learn to share, or one of the belligerents will loose

**Comment by MayamaMayama** [comment]
My Problem with Jekotia Ascendant ist that it imo doesnt feel fluent. It doesnt roll of the tongue. Im no native english speaker though so maybee its different for someone with english as his first language.

**Comment by RoyAwesome** [comment]
ya, I don't feel like Jekotia Ascendant works.  Jekotia Rising does.

**Comment by empZealoth** [comment]
I mean, Ascendant is a bit on a pretentious side, but rising might be a bit too generic.
One random thought i just had: Empires : Phoenix of Jekotia - we could spruce up that fugly NF flag with some sick ass red birb (phoenixes are usually depicted as red so it kinda passes NF vibe check) - and when you think of it, breakways/uprisings usually come with a solid heap of nationalistic mythology. I dunno if we have any artists, but a nice Phoenix design could serve as a logo, propaganda decals and stuff like that. 
"Like a Phoenix from the ashes, Jekotia rose once again"

**Comment by RoyAwesome** [comment]
The "Old Jekotian" flag is in Empires mod.  It's on the NF aircraft.  It's a bird.  I was thinking we can use it with the red line and stars from the Northern Faction to represent the new Jekotian Republic/Kingdom/Commonwealth/whatever faction.  

I don't actually mind a bit of generic in the name.  Generic is good if we can spice it up.  Generic with some spice is basically game naming 101, as it lets people remember, easy to google, and easy to get a good domain name for it (altho that last one isnt much of a concern for us).

**Comment by empZealoth** [comment]
Im not sure I agree with the generic part. Maybe its self confirmation bias, but when i try to think about it I really have trouble thinking of games I really like that would fit "generic with some spice"
Company of Heroes, Call of Doody, RED ORCHESTRA RISING STORM, MEDAL OF HONOR, etc are way more spicy than generic. Mount&Blade is pretty generic, but the subtitles are all spice. BANNERLORD is borderline english weebery to me.
KILLING FLOOR is pure spice. TOTAL WAR.
Some titles perfectly describe what the game is: Jump King (You're a KING, and, you JUMP), Space Engineers
Some generic names were part of genre defining games, so they kind of get a pass from me: The whole Metroidvania type, Minecraft (it describes the game perfectly though), Factorio too.
Lots of games have cryptic borderline nonsensical/alien/must-look-it-up, but unique names: Kenshi, Zero-k, Rust, V Rising.
To me Empires: Jekotia Rising is nothing but bland. Its like i opened google play store and found some shovelware knockoff of beloved IP (like when you search for Heroes of Might and Magic 3 and the first result you see is Heroes Magic War with a shamelessly ripped logo).
I think the name is a bit entwined with the lore, which i think needs to be more fleshed out. 
"Do you dare to fight for the unheard, or do you mistake order for peace?"

**Comment by yam4000** [comment]
empires: independance war

i believe such title will attract even people who werent looking for the game in the first place

**Comment by bloodwiper** [comment]
maybe we could go more abstract

Empires: Contested Crimson
Empires: Blemished Blue Reign
Empires: Hegemonic Cycle

**Comment by o-brkyr** [comment]
Mind that 99% of people who've played Empires didn't pay attention to, or were aware of, any lore for that game. I'm a fan of keeping it *really* simple and making sure there's a clear link to Empires mod - it doesn't need SEO. 

Of the ones listed, I'm a fan of 
- Fall of Empires
- Empires: New Dawn (Sounds a bit twilightly but it's also a very literal new dawn for the game)

Additional things I'd suggest are:
- Another Empire
- Rise of Empires
- Empires Ascending (*not* ascendant) or Empires Rising

 For what it's worth, I read "Jekotia" as "Jekosha", not "jekoteeah"

**Comment by slzimmer** [comment]
Empire's Mod

**Comment by jtg007** [comment]
Empires: A New Era

**Comment by empZealoth** [comment]
Empires: A New Era is kind of nice, but I'm worried it will be hard to find the game
Also, I agree "ascendant" is the wrong word, but i think
Empires: Jekotian Ascendancy would work better. There is literally ONE game with Ascendancy in it's title on steam. If you search for "rising" or "rise" or "rises" you get a zyllion pages of results

**Comment by Kaa4a** [comment]
1) REmpires
2) Empires: Future
3) Empires: Nothern/cold revenge
4) Empires: New War
5) Empires: Heroes VS Bstards
6) Empires: Final Words

**Comment by yam1917** [comment]
i think it would be a good idea to make a poll once enough ideas are collected / time is up

**Comment by Headshotmaster2006** [comment]
I'm going to throw this out there from a marketing standpoint. Make it _easy_, _simple_, and bridge a gap between the older die-hards and a new audience. 

Empires 2 or Empires:Remastered

My gut tells me "Empires 2". It worked for Dota 2, HL2, TF2, and every other direct/indirect sequels with a sequential number addon. It's not flashy, or prettied up, but it says "We did this, it's here,  it's new, and we've built upon something successful enough to warrant another." I would keep lore out of the title, such as faction names, because at first glance someone unfamiliar with the lore isn't going to get it, and might pass it over. Imagine if TF2 was named "Team Fortress: Blue's Revenge", or something like that. Does that roll off the tongue and intrigue you? Everything you throw at it from a marketing standpoint will have to sell both the game, and the name, if the name is complex. Especially if the the name sounds like an addon. There will be people out there that will think to themselves "Gee, that looks like DLC, and I don't have the base game."

I could be wrong though. You're best bet is doing a bunch of blind A/B testing. Get a bunch of names, put two side by side, and have people vote on which of the two. Then keep eliminating the ones that lose votes. Just an idea!

**Comment by Trainzack** [comment]
I think HSM's argument is convincing, I'm onboard with Empires 2.

**Comment by bloodwiper** [comment]
Isn't the entire point of this discussion is that we _cannot_ use the name "Empires 2"? Also would say that you can easily come up with counterexamples for HSM's argument (e.g. CSGO from CSS or Apex Legends being originally Titanfall 3). I do agree the name should be simple, and adding lore to the title might not do us favors. However, adding an abstract idea or attribute from the lore can make the title less generic and thus easier to find. It is a balancing act to pick a title that is memorable, rolls off the tongue, and inspires imagery but isn't convoluted.

It also would be worth trying to DMCA the fraudulent and false Empires currently using our title off of Steam.

**Comment by empZealoth** [comment]
We definitely need to find a name that is pretty unique because there is nothing worse for discoverability than literally not showing up in search results when you put the damn game name in

**Comment by empZealoth** [comment]
> I'm going to throw this out there from a marketing standpoint. Make it _easy_, _simple_, and bridge a gap between the older die-hards and a new audience.
> 
> Empires 2 or Empires:Remastered
> 
> My gut tells me "Empires 2". It worked for Dota 2, HL2, TF2, and every other direct/indirect sequels with a sequential number addon. It's not flashy, or prettied up, but it says "We did this, it's here, it's new, and we've built upon something successful enough to warrant another." I would keep lore out of the title, such as faction names, because at first glance someone unfamiliar with the lore isn't going to get it, and might pass it over. Imagine if TF2 was named "Team Fortress: Blue's Revenge", or something like that. Does that roll off the tongue and intrigue you? Everything you throw at it from a marketing standpoint will have to sell both the game, and the name, if the name is complex. Especially if the the name sounds like an addon. There will be people out there that will think to themselves "Gee, that looks like DLC, and I don't have the base game."
> 
> I could be wrong though. You're best bet is doing a bunch of blind A/B testing. Get a bunch of names, put two side by side, and have people vote on which of the two. Then keep eliminating the ones that lose votes. Just an idea!

Have you tried googling for empires? You get barely any results for actual empires, most of them are for other games with "empire" in the title, which is a gigantic problem in my opinion. Whenever i tried to convince any of my friends to try the game half of them would be turned off just by yhe fact it felt like i was trying to scam them

**Comment by Headshotmaster2006** [comment]
> Have you tried googling for empires? You get barely any results for actual empires, most of them are for other games with "empire" in the title, which is a gigantic problem in my opinion. Whenever i tried to convince any of my friends to try the game half of them would be turned off just by yhe fact it felt like i was trying to scam them

The biggest name we're up against is Age of empires in google(not counting the generic 'empires' search results). However, the search term 'Empires Mod' gives perfect results. Which tells me (our empires) presence is solid. I think if the name is generic, that's okay. With enough passive marketing and a decent steam release, it would get picked up on the search engine no problem.

I'm not saying my idea is perfect. Whatever is picked, it's needs to be simple, easy to market, memorable, and connects the first game with the 2nd. That's why my mindset is stuck on either "Empires 2", or 'remastered', 'resistance', etc...

Hell, I even think "Empires: No Surrender" would be pretty cool, but I'm pretty sure that wouldn't fly with _certain_ people.

To add to the conversation from the original post, I think the best name Roy presented was Empires:Northern Front. Just my 2 cents.

> Isn't the entire point of this discussion is that we cannot use the name "Empires 2"? Also would say that you can easily come up with counterexamples for HSM's argument (e.g. CSGO from CSS or Apex Legends being originally Titanfall 3). I do agree the name should be simple, and adding lore to the title might not do us favors. However, adding an abstract idea or attribute from the lore can make the title less generic and thus easier to find. It is a balancing act to pick a title that is memorable, rolls off the tongue, and inspires imagery but isn't convoluted.

I think we're still able to use Empires 2. No one has claimed it afaik, and it's so generic I doubt there would be any winning it on copyright grounds. You're right about CSGO, and CSS. I'm not married to Empires 2. I'm just married to the idea of simplicity and zero lore specific words in the tile. Especially because the lore, while cool, isn't the selling point. The lore is garnish. It looks pretty. But if the Empires dev team decided on a new IP, but kept core mechanics, the game would still be awesome.

**Comment by spritzles** [comment]
Regimes

**Comment by spritzles** [comment]
Empires: Territories

**Comment by TrueGuardian32** [comment]
Empirium: Conquest

**Comment by kilkkij** [comment]
Empires: Imperium in Imperio

It's about Jekotia establishing itself as a power within the greater Brenodi Empire.

**Comment by Wikonator** [comment]
"And the Stars wept for those Empires"

**Comment by BlackIceKB** [comment]
Empires: Dominion

**Comment by BlackIceKB** [comment]
Empires: Resurgence

**Comment by BlackIceKB** [comment]
Empires Escalation

**Comment by BlackIceKB** [comment]
Empires Reborn

Sorry if it is spam, you said 1 suggestion per comment :-)

**Comment by BlackIceKB** [comment]
An unconventional suggestion but -

Empires: Redivivus (latin for reborn)

**Comment by AlexWHite004** [comment]
ConglomeraT
Punkostrophe (an affinity of countries for self-degeneration or self-destruction, Stanislav Lem's term)
Dominanta
Val-Hal
Warriors of Fortune (Too much Deep Purple)
TimekillerZ

**Comment by spritzles** [comment]
Imperiums
